src/styles/blocks/main.scss

```diff
@@ -0,0 +1,8 @@
+.container {
+  display: grid;
+  padding: 50px 40px;
```

Comment:
The padding of 50px 40px might not be responsive on smaller screens. Consider using relative units or media queries to adjust the padding for different screen sizes to ensure a consistent layout.